components/color-palette/index.js

```diff
@@ -65,8 +65,17 @@ export default function ColorPalette( { colors, disableCustomColors = false, val
 					renderContent={ () => (
 						<ChromePicker
 							color={ value }
-							onChangeComplete={ ( color ) => onChange( color.hex ) }
-							disableAlpha
+							onChangeComplete={ ( color ) => {
+								let colorString;
+								if ( typeof color.rgb === 'undefined' || color.rgb.a === 1 ) {
+									colorString = color.hex;
+								} else {
+									const { r, g, b, a } = color.rgb;
+									colorString = 'rgba(' + [ r, g, b, a ].join( ',' ) + ')';
```

Comment:
You could use template strings here and omit the join; it'd probably be a bit more readable:

```js
colorString = `rgba(${ r }, ${ g }, ${ b }, ${ a })`;
```

I'm not sure why the `Array.join` is being used at all here, it just seems to make it more complex.

Reply:
@tofumatt yeah, I guess this is me still not 100% used to use ES6 features. But I totally agree with you. Just switched the variable to template strings.